contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -222,7 +222,7 @@ GetBulkLoadRequest(StringInfo message)
 							"unexpected token encountered processing the request.")));
 
 	offset++;
-
+	Assert(sizeof(uint16)<=sizeof(colCount));
```

Comment:
This check seems futile, its potentially checking "sizeof(uint16) <= sizeof(uint16)" which is always going to be true

Reply:
Yeah actually I was also thinking of the same but what if datatype of colCount is changed maybe in future

Reply:
If colcount data type changed then you are saying that the TDS format has changed and thus a new version of TDS would be required to be supported. Seems unlikely that We will see any update in format of message even if we have new versions of TDS released

Reply:
Okay